src/infrastructure/figma/types.ts

```diff
@@ -0,0 +1,5 @@
+export type WebhookPasscodeInput = {
```

Comment:
**suggestion:** Consider moving this type to `figma-service.ts` since it represents an utility type to describe the parameters of `FigmaService` method. We could even make this type private to that module.

Reply:
Now that we're generating a UUID for the passcode, this is no longer needed.